zebra-consensus/src/redjubjub/tests.rs

```diff
@@ -54,13 +58,17 @@ async fn batch_flushes_on_max_items() -> Result<()> {
     // flushing is happening based on hitting max_items.
     let verifier = Batch::new(Verifier::new(), 10, Duration::from_secs(1000));
     timeout(Duration::from_secs(5), sign_and_verify(verifier, 100))
-        .await
+        .await?
```

Comment:
This is what caused the warnings you ran into yesterday @hdevalence, it was originally only returning the error from the `Timeout` and discarding the `Result` from `sign_and_verify`.